### ISSUE TITLE
Sample data

### ISSUE THREAD

**Issue by sarath0993** [LeeThrzz/HCRB-MSAN-pytorch#2]
Can you provide sample data or data format to train a model?

**Comment by LeeThrzz** [comment]
Sorry, this is my first trying to upload code, so there are many problems in this repository. the training data format are .png or .tif(R G B bands),and the label data format is .png with sing band. I will provide sample data in future.

**Comment by sarath0993** [comment]
Ok sure, Thank you. If possible upload those text files contains list of images.
